### PR TITLE
BaSD-MS-70: Porfiles entity- CRUD

### PR DESCRIPTION
**Pull request by Micaela-89**
**Profiles Screens**

**List screen allows to:**

Visualize all profiles
Delete profiles
Click on any to update it
Click to add a new profile

**Form screen allows to:**

Create a new profile
Edit an existing profile
This will be updated in the list screen.

### CODE REVIEW

File: src/Components/Profiles/index.js [hunk 1]
```diff
@@ -1,9 +1,119 @@
...
 import styles from './profiles.module.css';
 
 function Profiles() {
+  const [showModal, setShowModal] = useState(false);
+  const [profiles, setProfiles] = useState([]);
+  const [selectedItem, setSelectedItem] = useState(undefined);
+  const [error, setError] = useState('');
+  const [isLoading, setLoading] = useState(false);
+
+  console.log(profiles);
```
Comment (estefrare) [inline src/Components/Profiles/index.js:12]: REMOVE THE CONSOLE.LOG

File: src/Components/Profiles/Form/form.module.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  margin-bottom: 30px;
+}
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  justify-content: center;
+}
+label {
```
Comment (estefrare) [inline src/Components/Profiles/Form/form.module.css]: do not use pure tags

File: src/Components/Profiles/Form/index.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
+import { useEffect, useState } from 'react';
+// import Select from '../Select';
+// import TextArea from '../TextArea';
```
Comment (estefrare) [inline src/Components/Profiles/Form/index.js]: remove comments

File: src/Components/Profiles/Modal/modal.module.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
+.container{
+  position:absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 120px;
+  width: 100%;
+  background-color: rgba(247, 233, 233, 0.5);
```
Comment (estefrare) [inline src/Components/Profiles/Modal/modal.module.css:8]: please, just use the styles I am using here https://github.com/BaSD-2021/Esteban-MindSet-app/pull/3/files#diff-ed523f7f05eb0244fb2951825b943d3c5a120bad0e90f877acee0683ff0ca377R1-R9

**Review by estefrare — CHANGES_REQUESTED** [review]

File: src/Components/Profiles/Form/index.js (hunk 3 above)
Comment (Micaela-89) [reply src/Components/Profiles/Form/index.js]: Already removed

File: src/Components/Profiles/Modal/modal.module.css (hunk 4 above)
Comment (Micaela-89) [reply src/Components/Profiles/Modal/modal.module.css:8]: Styles changed

**Review by estefrare — APPROVED** [review]

**Review by veronabaudracco — APPROVED** [review]

**Pull request merged**